pkgs/dart_model/test/scopes_test.dart

```diff
@@ -7,7 +7,7 @@ import 'package:dart_model/src/json_buffer/json_buffer_builder.dart';
 import 'package:test/test.dart';
 
 void main() {
-  group(Scope, () {
+  group('Scope', () {
```

Comment:
Note that I am converting these all to strings because anything else breaks the VsCode integration